cmd/installer/cli/reset.go

```diff
@@ -234,7 +241,14 @@ func checkErrPrompt(noPrompt bool, force bool, err error) bool {
 		return false
 	}
 	logrus.Info("Continuing may leave the cluster in an unexpected state.")
-	return prompts.New().Confirm("Do you want to continue anyway?", false)
+	confirmed, err := prompts.New().Confirm("Do you want to continue anyway?", false)
+	if errors.Is(err, terminal.InterruptErr) {
```

Comment:
why do we care about this type of error here if we're not reporting it?

Reply:
removed